lib/asyncbox.js

```diff
@@ -206,11 +205,12 @@ async function asyncfilter (coll, filter, runInParallel = true) {
  */
 async function waitForCondition (condFn, options = {}) {
   /** @type {WaitForConditionOptions & {waitMs: number, intervalMs: number}} */
-  const opts = _.defaults(options, {
+  const opts = {
```

Comment:
_.defaults work slightly differently. I don't think this is the exact replacement

Reply:
what's the difference between them?